projects/uitpas/reference/uitpas.json

```
@@ -2319,6 +2319,109 @@
       },
       "parameters": []
     },
+    "/passholders/me/checkin": {
+      "post": {
+        "summary": "Check-in current passholder",
```

```suggestion
        "summary": "Check the current passholder in",
```

Niet helemaal blij met mijn eigen suggestie, maar ik denk alleszins niet dat je "check-in" zo kan gebruiken als werkwoord

---

Check-in is een noun inderdaad, maar ik denk wel dat dit kan:
```suggestion
        "summary": "Check in the current passholder",
```

---

Klinkt goed! 👍